### ISSUE TITLE
Setting up SMET

### ISSUE THREAD

**Issue by UmangBR** [amosproj/amos2022ss03-turtlebot-fleet-management#13]
## User story
1. As a SD
2. I want to setup the Sick Map Engineering Tool
3. So that I can create maps for localization

## Acceptance criteria
* Licensing needs to be setup
* Should be able to start the software

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Sebastian2023** [comment]
![Sick_AMOS_SMET](https://user-images.githubusercontent.com/92729241/168662226-6f87305a-08a9-47f6-90d6-857ce7a75fc3.png)
